### PR TITLE
Replace tf.get_variable with tf.Variable

### LINKED ISSUE #414: tfa.text.crf_log_likelihood is not compatible with tf2.0-beta1
When implement CRF with tf2.0-beta1, got `AttributeError: module 'tensorflow' has no attribute 'get_variable'` when using tfa.text.crf_log_likelihood.
`AttributeError: in converted code:
    relative to /home/xingya:

    resume_parser/resume_parser/training/train_ner_bp.py:329 train_step  *
        log_likelihood, _ = tfa.text.crf_log_likelihood(
    .conda/envs/tf2/lib/python3.6/site-packages/tensorflow_addons/text/crf.py:191 crf_log_likelihood  *
        transition_params = tf.get_variable("transitions",
    .conda/envs/tf2/lib/python3.6/site-packages/tensorflow/python/autograph/impl/api.py:329 converted_call
        f = getattr(owner, f)

    AttributeError: module 'tensorflow' has no attribute 'get_variable'` 
Accordding to the example in the [official migration_guide](https://www.tensorflow.org/beta/guide/migration_guide), It should be tf.Variable.

### PR DESCRIPTION
**Pull request by Squadrick**
Closes #414

### CODE REVIEW

File: tensorflow_addons/text/crf.py [hunk 1]
```diff
@@ -188,8 +188,9 @@ def crf_log_likelihood(inputs,
 
     # Get the transition matrix if not provided.
     if transition_params is None:
-        transition_params = tf.get_variable("transitions",
-                                            [num_tags, num_tags])
+        initializer = tf.initializers.GlorotUniform()
```
Comment (seanpmorgan) [inline tensorflow_addons/text/crf.py]: Is there any precedent for using GlorotUniform here? Thanks very much for the PR as usual

Comment (facaiy) [reply tensorflow_addons/text/crf.py]: It's the default behavior for `get_variable`.
> https://www.tensorflow.org/api_docs/python/tf/get_variable?hl=en
> If initializer is None (the default), the default initializer passed in the variable scope will be used. If that one is None too, a glorot_uniform_initializer will be used.

Comment (Squadrick) [reply tensorflow_addons/text/crf.py]: This was why I used `GlorotUniform`, I could parameterize it with `GlorotUniform` being the default.

Comment (seanpmorgan) [reply tensorflow_addons/text/crf.py]: Hmmm I think parametizer-ing it may be a loverkill. This looks good to me.

**Pull request merged**